app/actions/websocket/index.ts

```diff
@@ -118,6 +119,7 @@ import {handleUserUpdatedEvent, handleUserTypingEvent, handleStatusChangedEvent}
 
 export async function handleFirstConnect(serverUrl: string) {
     registerDeviceToken(serverUrl);
+    setIgnoreNotificationACK(serverUrl);
```

Comment:
When is the first connect happen? Is this every time they open the app? 

Also, is there a way for the device to know when the notifications permissions have changed in the OS before the next connect?

Reply:
> When is the first connect happen? Is this every time they open the app?

Yes.

> Also, is there a way for the device to know when the notifications permissions have changed in the OS before the next connect?

No, as far as I know. I will investigate a bit further just in case.